### ISSUE TITLE
Suggestion: Server Mute people with mute command

### ISSUE THREAD

**Issue by michelhn** [jonas747/yagpdb#262]
It would be neat if people would be muted in voicechannels (server mute) if they get muted with the mute command:
![unknown](https://user-images.githubusercontent.com/31596904/40881436-f8c3bf3a-66c6-11e8-8bd9-ecc77fa93910.png)

**Comment by buthed010203** [comment]
Last I checked it does (have to set it up)

**Comment by michelhn** [comment]
Oh well, that's neat then I'll close this issue.

**Comment by michelhn** [comment]
After testing this: This doesn't work for me.

**Comment by buthed010203** [comment]
@michdi make sure its the same muted role, it should work as long as they dont have a role that force allows talking

**Comment by michelhn** [comment]
yeah I know that you can do this permission wise but server mute would be neat too
